conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -55,12 +55,9 @@ install:
     - cmd: conda install -c pelson/channel/development --yes --quiet obvious-ci
     {%- for channel in channels.get('sources', []) %}
     - cmd: conda config --add channels {{ channel }}{% endfor %}
-    - cmd: conda info
     - cmd: conda install -n root --quiet --yes conda-build anaconda-client jinja2 setuptools
```

Comment:
I think it does except for `setuptools`. Do we actually need that in the `root` environment? If not, I would happily remove this line.